packages/react-components/react-tag-picker-preview/src/components/TagPicker/TagPicker.cy.tsx

```diff
@@ -13,6 +13,18 @@ import { TagPickerOption } from '../TagPickerOption/TagPickerOption';
 import { Avatar } from '@fluentui/react-avatar';
 import { Button } from '@fluentui/react-button';
 
+/**
+ * This error means that ResizeObserver
+ * was not able to deliver all observations within a single animation frame.
+ * https://stackoverflow.com/questions/49384120/resizeobserver-loop-limit-exceeded
+ */
+const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/;
+Cypress.on('uncaught:exception', err => {
```

Comment:
why is this necessary?

Reply:
Cypress is failing due to some ResizeObserver loop error that can't be reproduced outside of cypress?!

Reply:
https://github.com/cypress-io/cypress/issues/20341

Reply:
can you link that issue in the comment too?